tests/tests_basic.yml

```diff
@@ -0,0 +1,99 @@
+- name: Basic snapshot test
+  hosts: all
+  vars:
+    - listA: []
+    - listB: []
+    - listC: []
+
+  tasks:
+
+    - name: Run the storage role to create test LVs
+      include_role:
+        name: fedora.linux_system_roles.storage
+
+    - name: Get unused disks
+      include_tasks: get_unused_disk.yml
+      vars:
+        min_size: "1g"
+        min_return: 10
+
+    - block:
+        - debug:
+            var: listA
+        - debug:
+            var: listB
+        - debug:
+            var: listC
+        - set_fact:
+            listA: "{{ listA + [ unused_disks[item|int] ] }}"
+          with_sequence: start=0 end=2 stride=1
+        - set_fact:
+            listB: "{{ listB + [ unused_disks[item|int] ] }}"
+          with_sequence: start=3 end=5 stride=1
+        - set_fact:
+            listC: "{{ listC + [ unused_disks[item|int] ] }}"
+          with_sequence: start=6 end=9 stride=1
+
+    - block:
+        - debug:
+            var: listA
+        - debug:
+            var: listB
+        - debug:
+            var: listC
+
+    - name: Create LVM logical volumes under volume groups
+      include_role:
+        name: linux-system-roles.storage
+      vars:
+        storage_pools:
+          - name: test_vg1
+            disks: "{{ listA }}"
+            volumes:
+              - name: lv1
+                size: "15%"
+
+              - name: lv2
+                size: "50%"
+
+          - name: test_vg2
+            disks: "{{ listB }}"
+            volumes:
+              - name: lv3
+                size: "10%"
+
+              - name: lv4
+                size: "20%"
+
+          - name: test_vg3
+            disks: "{{ listC }}"
+            volumes:
+              - name: lv5
+                size: "30%"
+
+              - name: lv6
+                size: "25%"
+
+              - name: lv7
+                size: "10%"
+
+              - name: lv8
+                size: "10%"
+
+    - name: Run the snapshot role to create snapshot LVs
+      include_role:
+        name: linux-system-roles.snapshot
+      vars:
+        snapshot_lvm_percent_space_required: 15
+        snapshot_all: true
+        snapshot_lvm_suffix: _z
+        snapshot_lvm_prefix: a_
+        snapshot_lvm_action: snapshot
+
+    - name: Run the snapshot role remove the snapshot LVs
+      include_role:
+        name: linux-system-roles.snapshot
+      vars:
+        snapshot_lvm_suffix: _z
+        snapshot_lvm_prefix: a_
+        snapshot_lvm_action: clean
```

Comment:
should have a following task to verify snapshots have been cleaned

Reply:
Ok - added verify option to clean command.